src/main.cpp

```diff
@@ -127,6 +128,9 @@ static bool _get_options(int argc, char **argv, Options &options)
 		("start-daemon", po::value<bool>(&options.start_daemon)->default_value(true),
 		 "Start the aktualizr-lite systemd service automatically after performing the registration.")
 
+		("use-proxy", po::value<bool>(&options.use_proxy)->default_value(false),
```

Comment:
probably should default to true or we'll never get people on this

Reply:
done